SuiteCRM/SuiteCRM
# add option to run workflow multiple runs once a period of time

Today workflow repeated runs can set as only on or off. 

This PR adds more capabilities to trigger repeated run on period of time.

## Description





Lot of business logic require repeated runs but not each minute as it's existing today. Running workflow action once a hour, day or month, is very common scenario. Today to acheive this task need to write a cron job which require code skills.

## Motivation and Context

This PR give ability to run workflow on repeated run hourly, daily, weekly, monthly or annually, in addition to the old modes - on [minutely]  & off.

![image](https://user-images.githubusercontent.com/1040582/88550562-75190800-d02a-11ea-9a79-0fe7273396e2.png)

## How To Test This

Need to configure each multiple run option (old & new) and check it's behave like expected. For long period, need to fake the environment datetime..

## Types of changes


### Final checklist
